### ISSUE TITLE
Insufficient async-kit version specified

### ISSUE THREAD

**Issue by JetForMe** [vapor/fluent-kit#384]
It seems fluent-kit specifies async-kit 1.0.0, but needs something more recent. I got this error in Xcode with version 1.1.1:

```
.../Xcode/DerivedData/camsync-bylosnkysgfpvnbwbikdmasqdlgw/SourcePackages/checkouts/fluent-mysql-driver/Sources/FluentMySQLDriver/FluentMySQLConfiguration.swift:130:34: Extra argument 'requestTimeout' in call

        let pool = EventLoopGroupConnectionPool(
            source: db,
            maxConnectionsPerEventLoop: self.maxConnectionsPerEventLoop,
            requestTimeout: self.connectionPoolTimeout,
            on: databases.eventLoopGroup
        )
```

**Comment by gwynne** [comment]
This should be updated in https://github.com/vapor/vapor.git as well.

**Comment by gwynne** [comment]
This was incidentally addressed in #453.
